README.md

```diff
@@ -70,62 +75,39 @@ $ diesel migration run
 
-Configs for the specified network are in the `--home-dir` provided folder. We need to ensure that NEAR Indexer for Explorer follows
-all the necessary shards, so `"tracked_shards"` parameters in `~/.near/testnet/config.json` needs to be configured properly.
-For example, with a single shared network, you just add the shard #0 to the list:
+Unlike the original [NEAR Indexer for Explorer](https://github.com/near/near-indexer-for-explorer) you **can't** tell Indexer to store data from genesis (Accounts and Access Keys) by adding key `--store-genesis` to the `run` command. So please, ensure you took care about the genesis data in your database in order this indexer to work properly.
 
 ```
-...
-"tracked_shards": [0],
-...
+TODO: Implement a tool to fill the database with the genesis data
 ```
 
-### Run NEAR Indexer for Explorer
-
-Command to run NEAR Indexer for Explorer have to contain sync mode.
-
-You can choose NEAR Indexer for Explorer sync mode by setting what to stream:
- - `sync-from-latest` - start indexing blocks from the latest finalized block
- - `sync-from-interruption --delta <number_of_blocks>` - start indexing blocks from the block NEAR Indexer was interrupted last time but earlier for `<number_of_blocks>` if provided
- - `sync-from-block --height <block_height>` - start indexing blocks from the specific block height
-
-Optionally you can tell Indexer to store data from genesis (Accounts and Access Keys) by adding key `--store-genesis` to the `run` command.
-
-NEAR Indexer for Explorer works in strict mode by default, but you can disable it for specific amount of blocks. The strict mode means that every piece of data
-will be retried to store to database in case of error. Errors may occur when the parent piece of data is still processed but the child piece is already
-trying to be stored. So Indexer keeps retrying to store the data until success. However if you're running Indexer not from the genesis it is possible that you
-really miss some of parent data and it'll be impossible to store child one, so you can disable strict mode for 1000 blocks to ensure you've passed the strong
-relation data area and you're running Indexer where it is impossible to loose any piece of data.
+NEAR Indexer for Explorer works in strict mode by default, but you can disable it. The strict mode means that every piece of data
+will be retried to store to database in case of error. Errors may occur when the parent piece of data is still processed but the child piece is already trying to be stored. So Indexer keeps retrying to store the data until success. However, if you're running Indexer not from the genesis it is possible that you really miss some of parent data and it'll be impossible to store a child one, so you can disable strict mode to ensure you've passed the strong relation data area and you're running Indexer where it is impossible to loose any piece of data.
```

Comment:
I wish some native speakers rephrase this paragraph to be understandable. I guess I've written it and I don't get a thing of it :(

Reply:
I tried my best, how's this?

---

NEAR Indexer for Explorer works in strict mode by default. In strict mode, the Indexer will ensure parent data exists before storing children, infinitely retrying until this condition is met. This is necessary as a parent (i.e. `block`) may still be processing while a child (i.e. `receipt`) is ready to be stored. This scenario will likely occur if you have not stored the genesis file or do not have all data prior to the block you start indexing from. In this case, you can disable strict mode to store data prior to the block you are concerned about, and then re-enable it once you have passed this block.

To disable strict mode provide the following command arugment:

```
--non-strict-mode
```

Reply:
I've already updated the `README.md` to include this, let me know if you're not happy with it.

Also including subheadings in this section to make things clearer.

Reply:
Yes! Thank you! Much better that I've written for sure :)